test/unit/api-test.js

```diff
@@ -58,43 +65,42 @@ describe('Api', function() {
         assert.equal(check, false, 'api.off works');
     });
 
-    it('bad events don\'t break player', function() {
-        jwplayer.debug = false;
+    it('bad events doesn\'t break player', function() {
```

Comment:
Better yet, let's rename the test "event handlers that throw exceptions to not stop event propagation".

(Setting `jwplayer.debug === true` is the only way to disable catching these exceptions)